procedures/release-duty/merge-duty/merge_new_esr_branch.rst

```diff
@@ -111,10 +111,18 @@ Task list and known dependencies
 
 22. Before gtb for XX.3.0 (beginning of RC week for XX+3), update balrog rules
     on esr-localtest and esr-cdntest to allow updates to esrXX; check rules on
-    the release channel 
+    the release channel, and check with release management for any necessary
+    watershed and/or desupport rules.
 
-23. Some time later (maybe soon after the XX.3.0 release to avoid forgetting),
+23. Shortly before the XX.3.0 release, update shipit's `CURRENT_ESR` and `ESR_NEXT` config variables.
```

Comment:
Maybe this should be "shortly after", so that we control the timing of XX.3.0 showing up as current.

Reply:
The reason I said shortly before is, once this is deployed, the following product-details rebuild will cause things like www.mozilla.org to only show the new release, which we probably don't want to have happen before XX.3.0 is officially out.  Maybe it should say "at XX.3.0 release time" instead.